deploy/docker/utils/bin/backup.js

```diff
@@ -21,7 +26,14 @@ async function run() {
 
     utils.stop(['backend', 'rts']);
 
-    const timestamp = new Date().toISOString().replace(/:/g, '-')
+    console.log('Available free space at /appsmith-stacks');
+    const availSpaceInBytes = Number(shell.exec('df --output=avail -B 1 /appsmith-stacks | tail -n 1'));
```

Comment:
Minor, it's recommended to use `parseInt` to turn strings into Number in JS, unless if there's a specific reason not to.

```suggestion
    const availSpaceInBytes = parseInt(shell.exec('df --output=avail -B 1 /appsmith-stacks | tail -n 1'), 10);
```